release-please-config.json

```diff
@@ -8,7 +8,8 @@
       "bump-minor-pre-major": true,
       "bump-patch-for-minor-pre-major": true,
       "extra-files": [
-        "README.md"
+        "README.md",
+        "server.json"
```

Comment:
I think you'll need to use the arbitrary JSON configuration for this to work properly.

https://github.com/googleapis/release-please/blob/main/docs/customizing.md#updating-arbitrary-json-files